crypto/fipsmodule/kdf/internal.h

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#include <openssl/digest.h>
+#include <openssl/hmac.h>
+
+typedef struct {
+  void *data;
```

Comment:
I guess this is `void *` because it can point to `SSKDF_VARIANT_DIGEST_CTX` or `SSKDF_VARIANT_HMAC_CTX`?

Reply:
That is correct!